### DISCUSSION TITLE
simbot-mirai-login-helper如何使用？

### DISCUSSION THREAD

**Discussion by codez-yydd** [simple-robot/Discussions#134] [Q&A]
### 涉及的编程语言

Java

### 涉及的组件库

mirai (QQ)

### 疑问描述

simbot-mirai-login-helper这个是如何使用的？我管理员运行点击验证后会提示报错 显示本地一个地址 java版本 浏览器版本 win版本 之类的信息 浏览器也会被打开，软件会显示准备登陆BOT就没有任何反应了

**Comment by codez-yydd** [comment]
Unable to establish websocket connection to http:/lcalost5317/devtoolsbrowser/17ed4566 2154309 -87f 519c81eeSfb3 Build info: version: '4.1.2', revision: '9a5a329c5a'

System info: host: 'DESKTOP 9VELVCP', ip: 192.168.0.186', os.name: 'Windows 11, os.arch: 'amd64', os.version: '10.0 java.version: 17.0.2 Driver info; driver.version: EdgeDriver

**Reply by ForteScarlet** [reply]
这个很久没更新了，内部的mirai版本比较旧了有可能已经用不了了

**Reply by codez-yydd** [reply]
账号风控无法解决了吗，我已经引入了FixProtocolVersion这个包，正常运行了1个月左右的样子，现在无法登陆了，报的是45 qq版本过低

**Reply by ForteScarlet** [reply]
FixProtocolVersion版本至少在 v1.4.0+，并且设备类型为 PAD 相关的类型，如果依旧不行那可能就需要更换账号或等待mirai更新 v2.15
